### DISCUSSION TITLE
Netbox Topology - Coordinate

### DISCUSSION THREAD

**Discussion by Mylevsky** [mattieserver/netbox-topology-views#479] [Q&A]
Good afternoon! Could you please tell me why all my devices are pulled to the center? Is it possible to make them centered around the device, as in the mode without "save coordinate"?
![1](https://github.com/netbox-community/netbox-topology-views/assets/64000624/1eb9f7d0-0ac0-4907-9106-4fafc4729bb9)
![2](https://github.com/netbox-community/netbox-topology-views/assets/64000624/0aa0fbb2-a378-4e8a-99d3-d7180bb7474f)

**Comment by dreng** [comment]
Hi,

the checkbox "Save Coordinates" does not place any devices nor does it the opposite. It only provides that positions of dragged devices will be stored into the database.

If the database hasn't got any coordinate information about a device, the physics engine places it wherever it thinks it's a good idea. If the checkbox "Save Coordinates" is not activated, dragged devices will jump back to it's origin because even after dragging the icon, there's no information about its coordinates and the physics engine does its job.

The only thing you can do to prevent this: Turn on "Save Coordinates" and drag devices where you want them to be.
